packages/server-renderer/src/render.ts

```diff
@@ -73,6 +73,7 @@ export function createBuffer() {
       const isStringItem = isString(item)
       if (appendable && isStringItem) {
         buffer[buffer.length - 1] += item as string
+        return
       } else {
```

Comment:
Could remove this `else` altogether with the early return

Reply:
Done. I originally did, but it had a strange effect on synthetics, maybe some JIT optimizations can optimize branching work differently in this case. Fortunately this is more often good than bad.